From 9c33c4875fbbcb278e3610701cc4eb35e0abf90d Mon Sep 17 00:00:00 2001
From: Zhao Qiang <B45475@freescale.com>
Date: Wed, 17 Sep 2014 15:34:17 +0800
Subject: [PATCH 27/57] t1040rdb: add slic tdm node to t1040rdb

add slic tdm node into t1040rdb.dts

Signed-off-by: Zhao Qiang <B45475@freescale.com>
Change-Id: I1862ebef1b7bfd5a43258afbd62bcfe417dc61c0
Reviewed-on: http://git.am.freescale.net:8181/19000
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Xiaobo Xie <X.Xie@freescale.com>
Reviewed-by: Richard Schmitt <richard.schmitt@freescale.com>
[Original patch taken from QorIQ-SDK-V1.8-20150619-yocto.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/t1040rdb.dts |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/t1040rdb.dts b/arch/powerpc/boot/dts/t1040rdb.dts
index 5d9f5c0..5695fbc 100644
--- a/arch/powerpc/boot/dts/t1040rdb.dts
+++ b/arch/powerpc/boot/dts/t1040rdb.dts
@@ -153,6 +153,12 @@
 				reg = <2>;
 				spi-max-frequency = <8000000>;
 			};
+
+			slic@3 {
+				compatible = "maxim,ds26522";
+				reg = <3>;
+				spi-max-frequency = <2000000>; /* input clock */
+			};
 		};
 
 		fman0: fman@400000 {
-- 
1.7.5.4

